BLSQ/iaso
# IA-1930: add missing french translations in DataSourcesDialog

Missing french translations for test settings success/error + projects input label

Related JIRA tickets : [IA-1930](https://bluesquare.atlassian.net/browse/IA-1930?atlOrigin=eyJpIjoiNWZhNDE2Nzg3YzZhNDkxNzk3NTdhMzI2N2YzM2YyZWYiLCJwIjoiaiJ9)

## Self proofreading checklist


## Changes

- add missing french translations
- small refactor using formatMessage instead of FormattedMessage component

## How to test

- Go to Orgunits 
- Datasources list
- Click on create
- Play with dhis2 fields to see either success or error message below "Test Settings"

## Print screen / video
<img width="1429" alt="Capture d’écran 2023-02-21 à 12 20 12" src="https://user-images.githubusercontent.com/25134301/220331501-15f68a8b-6590-4739-935a-26d6ffac838c.png">


[IA-1930]: https://bluesquare.atlassian.net/browse/IA-1930?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
